src/analysis/import/import_osm.sh

```diff
@@ -207,6 +207,27 @@ else
     echo "The city residential default speed is ${CITY_DEFAULT}."
 fi
 
+# Save default speed limit to a table for export later
+psql -h $NB_POSTGRESQL_HOST -U $NB_POSTGRESQL_USER -d $NB_POSTGRESQL_DB \
+     -c "CREATE TABLE IF NOT EXISTS \"residential_speed_limit\" (
+            fips_code_state char(2),
+            fips_code_city char(7),
+            state_speed smallint,
+            city_speed smallint
+        );"
+psql -h $NB_POSTGRESQL_HOST -U $NB_POSTGRESQL_USER -d $NB_POSTGRESQL_DB \
+     -c "INSERT INTO \"residential_speed_limit\" (
+            state_fips_code,
+            city_fips_code,
```

Comment:
These two lines are throwing an error for me currently when trying to run a new analysis - looks like a minor typo. 

Should be `fips_code_state` and `fips_code_city` instead of `state_fips_code` and `city_fips_code`

Reply:
I changed my mind about these names at the last minute and apparently updated it incompletely then didn't re-check.  These are the names I wanted to switch to, so I just changed the ones above that I missed.  And ran another analysis, which has gotten past this point.